plugins/postgres/dbt/adapters/postgres/relation_configs/materialized_view.py

```diff
@@ -36,7 +37,6 @@ class PostgresMaterializedViewConfig(RelationConfigBase, RelationConfigValidatio
     """
 
     table_name: str = ""
-    query: str = ""
```

Comment:
This was not used, and is the only difference in this context between a ModelNode and a ParsedNode